index.html

```diff
@@ -4,7 +4,8 @@
 		<meta charset="UTF-8" />
 		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
 		<title>Apto PCI SDK interactive example</title>
-		<script src="https://cdn.jsdelivr.net/npm/@apto-payments/pci-sdk-web@2"></script>
+		<script src="./packages//pci-sdk-web/dist/umd/apto-pci-sdk.js"></script>
+		<!-- <script src="https://cdn.jsdelivr.net/npm/@apto-payments/pci-sdk-web@2"></script> -->
```

Comment:
Pointing this out because I really don’t know 😅  Do you want to commit this? Or was this to work locally? 🤔